site/src/pages/TasksPage/TasksPage.tsx

```diff
@@ -77,7 +77,7 @@ const TasksPage: FC = () => {
 						onRetry={aiTemplatesQuery.refetch}
 					/>
 					{aiTemplatesQuery.isSuccess && (
-						<section>
+						<section className="pt-8 pb-8">
```

Comment:
```suggestion
						<section className="py-8">
```

jsyk, there are shorthands for "padding along the y-axis on both ends" and "padding along the x-axis on both ends"